### ISSUE TITLE
Unable to add members to an org in the UI

### ISSUE THREAD

**Issue by paynejd** [OpenConceptLab/ocl_issues#1022]
The "Edit Membership" panel only loads a subset of the users so it is not possible to add most users to an org. I had to do this via the API instead (which was fine for now).

We need to use different approaches for adding and removing users.

## Adding a user
To add a user, we need to populate the dropdown dynamically as a user types. Ideally, we could find users by username, email or full name, same as GitHub -- though it's fine to stick with just username for now. See this screenshot from github:
![image.png](https://images.zenhubusercontent.com/5981a15ff8b79f8376e1a78e/6f9ff62c-bb4c-4419-a91f-916850abaa41)

## Removing a user
To remove a user, let's move that inline in the list of users -- i.e. an Actions menu using the 3 dots, similar to other places in the TermBrowser. GitHub also does something similar:
![image.png](https://images.zenhubusercontent.com/5981a15ff8b79f8376e1a78e/be2d2a07-8cf7-4a20-9c3a-195c4a9fbe9e)

**Comment by jamlung-ri** [comment]
This ticket should also allow us to Edit organization membership from the individual user's page too. This work was started but not completed in #601.

**Comment by ManojLL** [comment]
shall I work on this?

**Comment by snyaggarwal** [comment]
@ManojLL Yes, you can pick this up and raise PR. Do let us know here if you need any directions.

**Comment by jamlung-ri** [comment]
Outdated ticket - relevant ideas have been represented in OCL KB

**Comment by jamlung-ri** [comment]
Outdated ticket - relevant ideas have been represented in OCL KB
